student.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id, :parent_permission
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_service?
+    is_of_age? || @parent_permission
+  end
```

Comment:
- [ ] `can_use_service?` is not a private method. It should be placed before the private keyword.